Libplanet/Store/Trie/MerkleTrieExtensions.cs

```diff
@@ -0,0 +1,70 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet.Store.Trie.Nodes;
+
+namespace Libplanet.Store.Trie
+{
+    /// <summary>
+    /// An extension class to provide some convenient methods to deal with <see cref="MerkleTrie"/>.
+    /// </summary>
+    public static class MerkleTrieExtensions
+    {
+        /// <summary>
+        /// Compare two given tries and get different value nodes (leaf node).
```

Comment:
.NET docs apparently prefer third-person singular verbs.  Here's an [unofficial guide for docs style][1]:

> Consider using third person singular verb to start the summary of members.

```suggestion
        /// Compares two given tries and gets different value nodes (leaf node).
```

[1]: https://weblogs.asp.net/dixin/csharp-coding-guidelines-6-documentation